build_tools/github/trigger_hosting.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -x
+
+GITHUB_RUN_URL=https://nightly.link/$GITHUB_REPOSITORY/actions/runs/$RUN_ID
+BRANCH=$( wget $GITHUB_RUN_URL/branch.zip && \
```

Comment:
Can we get the branch name from the `github` object? There seems to be alot of information that is available in `workflow_run`: https://github.com/braindecode/braindecode/blob/1242235950dcd23d651fd13c49b52682d635994d/.github/workflows/push-artifact-to-circleci.yml#L20-L25

Also do we need the branch name? Is it enough to have the PR number: https://github.com/braindecode/braindecode/blob/1242235950dcd23d651fd13c49b52682d635994d/.github/workflows/push-artifact-to-circleci.yml#L44

Reply:
> Can we get the branch name from the `github` object? There seems to be alot of information that is available in `workflow_run`: https://github.com/braindecode/braindecode/blob/1242235950dcd23d651fd13c49b52682d635994d/.github/workflows/push-artifact-to-circleci.yml#L20-L25
> 

I will have a look again to the context of the `workflow_run`  in the `github` object, but, IIRC, I didn't any interesting in there.

> Also do we need the branch name? Is it enough to have the PR number: https://github.com/braindecode/braindecode/blob/1242235950dcd23d651fd13c49b52682d635994d/.github/workflows/push-artifact-to-circleci.yml#L44

If we want this PR to be executed in the `main` and releases branches, I would say that we need the branch, right?